tools/wptrunner/wptrunner/testloader.py

```diff
@@ -489,6 +493,9 @@ def make_queue(cls, tests, **kwargs):
 
             test_queue.put((group, group_metadata))
 
+        for _ in range(kwargs["processes"]):
```

Comment:
Having to implement this in every derived class doesn't seem ideal; it would be better to have a single implementation that's always used.